bluebrain/repo-bluebrain/packages/py-archngv/package.py

```diff
@@ -13,17 +13,15 @@ class PyArchngv(PythonPackage):
     git = "ssh://git@bbpgitlab.epfl.ch/nse/ArchNGV.git"
 
     version("develop", branch="main")
-    version("2.0.2", tag="ArchNGV-v2.0.2")
-    version("2.0.1", tag="ArchNGV-v2.0.1")
-    version("3.0.0", tag="ArchNGV-v3.0.0")
+    version("3.0.1", tag="ArchNGV-v3.0.1")
 
     depends_on("py-setuptools@42:", type="build")
 
     depends_on("py-numpy@1.19.5:", type=("build", "run"))
```

Comment:
Maybe unrelated to this MR, but it should be `numpy<=1.22` according to setup.py